config/vendor_api/draft.yml

```diff
@@ -154,591 +55,32 @@ paths:
-        '401':
-          "$ref": "#/components/responses/Unauthorized"
-        '404':
-          "$ref": "#/components/responses/NotFound"
-        '422':
-          "$ref": "#/components/responses/UnprocessableEntity"
-  "/applications/{application_id}/interviews/{interview_id}/cancel":
-    post:
+  "/reference-data/rejection-reason-codes":
+    get:
       tags:
-      - Interview management
-      summary: Cancel interview
-      description: |
-        Cancels an existing interview for this application.
-      parameters:
-      - "$ref": "#/components/parameters/application_id"
-      - "$ref": "#/components/parameters/interview_id"
-      requestBody:
-        description: Interview details
-        content:
-          application/json:
-            schema:
-              type: object
-              required:
-                - data
-                - meta
-              properties:
-                data:
-                  "$ref": "#/components/schemas/CancelInterview"
-                meta:
-                  "$ref": "#/components/schemas/MetaData"
+      - Reference data
       responses:
         '200':
-          description: An application
+          description: An array of possible rejection reason codes and rejection reason descriptions.
           content:
             application/json:
               schema:
-                "$ref": "#/components/schemas/SingleApplicationResponse"
-        '401':
-          "$ref": "#/components/responses/Unauthorized"
-        '404':
-          "$ref": "#/components/responses/NotFound"
-        '422':
-          "$ref": "#/components/responses/UnprocessableEntity"
-  "/applications/{application_id}/notes/create":
-    post:
-      tags:
-      - Note management
-      summary: Create note
-      description: |
-        Creates a new note for this application.
-        The `updated_at` timestamp will be set to the time of the API call.
-        Its author details will be read from the attribution part of MetaData.
-      parameters:
-      - "$ref": "#/components/parameters/application_id"
-      requestBody:
-        description: Note details
-        content:
-          application/json:
-            schema:
-              type: object
-              required:
-                - meta
-                - data
-              properties:
-                data:
-                  "$ref": "#/components/schemas/CreateNote"
-                meta:
-                  "$ref": "#/components/schemas/MetaData"
-      responses:
-        '200':
-          description: An application
-          content:
-            application/json:
-              schema:
-                "$ref": "#/components/schemas/SingleApplicationResponse"
-        '401':
-          "$ref": "#/components/responses/Unauthorized"
-        '404':
-          "$ref": "#/components/responses/NotFound"
-        '422':
-          "$ref": "#/components/responses/UnprocessableEntity"
+                "$ref": "#/components/schemas/ListResponse"
```

Comment:
presumably the response will be a hash of codes and labels, such as 
```
[{\"code\":\"A1\",\"label\":\"Nah mate\"},{\"code\":\"A2\",\"label\":\"Not a good vibe\"}]
```
is this covered by `ListResponse`?

Reply:
Possibly the simpler and definitely lazier approach was to return something like

```
[
  "A1: Nah mate",
  "A2: Not a good vibe"
]
```